src/python/grpcio_status/grpc_status/rpc_status.py

```diff
@@ -30,7 +31,7 @@ class _Status(
     pass
 
 
-def from_call(call):
+def from_call(call: grpc.Call) -> Optional[status_pb2.Status]:
```

Comment:
We should add something to doc string mentioning what it means to return `None`.